tests/test_cleaneval.py

```diff
@@ -0,0 +1,486 @@
+#!/usr/bin/env python2.5
+# encoding: utf-8
+"""
+cleaneval.py v1.0
+
+Simple and fast evaluation of CleanEval-1 tasks (precision, recall, F-score).
+"""
+
+import time
+import sys
+import os
+import string
+import chardet
+import codecs
+import re
+import requests
+sys.path.append('')
+import metahtml
+import trafilatura
+from newspaper import parsers
+from bs4 import BeautifulSoup
+from glob import glob
+from math import *
+from difflib import SequenceMatcher
+from getopt import getopt, GetoptError
+from multiset import Multiset
+
+help_message = '''
+This is cleaneval.py version 1.0 -- Copyright (C) 2008 by Stefan Evert
```

Comment:
This help message is no longer applicable; you need to write your own

Reply:
It doesn't look like any of the code in this file can be called from the command line as-is;

See https://github.com/mikeizbicki/metahtml/blob/f60ece260fbb0b33e23cb2515b406cd50c58fd9c/tests/test_content.py#L279 for a method for converting a function into a command-line utility

Reply:
as part of the transformation mentioned above, this help message will be put in the docstring of your main function